### DISCUSSION TITLE
personal-development-mehr-1403/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#57] [Announcements]
# personal-development-mehr-1403/

ماه مهر هم به انتهای خودش رسید و مثل همیشه پر از چالش و یادگیری بود اما این دفعه از جنس مثبت.
دقیقاً‌ پارسال همین مواقع بود که داشتم برای دوره آموزشی سربازی آماده می‌شدم. الحق و الانصاف که دوره عبثی بود و سرتاسر طراحی شده بود که وقت آدم تلف بشه. هر چی بود تموم شد.
این ماه اکثراً روی مطالعه کتاب‌ حول استراتژی و حل مسئله و این‌ها گذشت تا این که بصورت کاملاً اتفاقی با سیستم‌های مدیریت زندگی دیجیتال (و مدیریت دانش) آشنا شدم که جلوتر توضیح میدم.

https://aprd.ir/personal-development-mehr-1403/

**Comment by elhamghadiri** [comment]
سلام آقای پورمند
مطالبی که می‌نویسید به شدت مفید و آموزنده و در عین حال جذاب هستند. 
مثلا در همین نوشته هم سیستم نوت برداری خیلی جالب بود برام (چون همیشه به دنبال نحوه نوت برداری و دسته‌بندی و لینک نوت‌هام هستم)، هم مطالبی که راجع به تغذیه و ورزش مینویسید (در این مورد هم من همچنان پیگیر یادگیری و توجه به بدنم هستم) و در کل همیشه از خوندن مطالبی که مینویسید لذت میبرم.
موضوع دیگه که دوست داشتم اشاره کنم اینکه دیروز که سرکار بودم دیدم مطلب جدید نوشتید اما راستش دلم نیومد هول هولی بخونم و امروز که خونه هستم همزمان با نوشیدن قهوه صبحگاهی این مطلب جذاب رو هم خوندم. 

در آخر هم بگم که اون قضیه اینکه فکر میکنید غرور خاصی در نوشته‌هاتون هست رو من اصلا اینطور برداشت نکردم و به نظرم خیلی دست و دلبازانه هرچیزی که یاد میگیرید رو با منبع و لینک عنوان میکنید و این برای من خواننده خیلی ارزشمنده.

**Comment by mrahar** [comment]
سلام امیر جان وقتت بخیر.
مطلبی که در مورد زتلکستن نوشتی برام خیلی مفید بود. دوره یودمی که معرفی کردی دیدم، دوره خوبی بود.
در ویدئو چند تا توصیه داشت که با مطالبی که قبلا خونده بودم در یک راستا بود. 

اولیش مربوط به نوشتن نت های fleeting(نت های کوتاه برای یادآوری و وارد کردن نت اصلی) بود، که نهایتا باید بعد  از یک روز  به زتلکستن(محل اصلی ذخیره نت ها) خودمون باید وارد کنیم. در این مورد یک ویدئو از اندرو هوبرمن با عنوان Optimal Protocols for Studying & Learning دیده بودم که توضیح داده بود اگر مطلبی بعد از *یک روز* از خودمون بپرسیم(self evaluation)، پنجاه درصد عملکرد یادگیری بهبود می یابد. البته یادگیری قاعدتا در اینجا در حد مرحله اول از طبقه بندی بلوم است و بیشتر به حفظ کردن ربط دارد. 

همچنین همین که مطلبی از زبان خودمان دوباره به زتلکستن وارد کنیم، داریم اولین مرحله درک مطلب بلوم را انجام می دهیم. در متمم یاد گرفتیم اولین مرحله درک مطلب ترجمه است. اینکه مطالب با زبان و کلمات خودمان بیان کنیم.

 در مرحله بعدی با لینک کردن مطالب به همدیگه تا مرحله بعدی بلوم یعنی تفسیر و ربط دادن مطالب بهم جلو می رویم. پس خود روش زتلکستن اگر درست اجرا شود تا درک مطلب به ما کمک میکند که واقعا خیلی اثر گذار است. 

البته من خودم هم همیشه نت هامو از زبان خودم و با concept map سعی در تفسیر اونها داشتم اما اینکه تمام نت ها در یک برنامه جلوی چشمم باشه خیلی خیلی راحتر هستم. 
این برنامه به نوعی شبیه نقشه گوگل مپ است، اینکه هرجا خواستیم زوم میکنیم کانتکس میبینم و دوباره زوم اوت میکنیم.

در ویدئو چند بار به عبارت سرندیپی اشاره شد، البته گوینده ویدئو گفتند سرندیپیتی اما محمدرضا در کتاب از کتاب توضیح داد سرندیپی یعنی منسوب به سرندیپ. و معنی دستاورد تصادفی در تلاش آگاهانه می دهد که دقیقا در استفاده از روش زتلکستن صدق میکند.

من هم دیگه مثل شما باید نت هامو به اوبسیدین منتقل کنم، مقداری در مورد استفاده از فولدر، تگ دچار نفهمی شدم ممنون میشم شما توضیح بدین چطوری برای نت برداری از متمم از فولدر تگ استفاده می کنید. 

با تشکر از معرفی این روش ارزشمند 
ارادتمند
محمدرضا نظرآهاری

**Reply by pourmand1376** [reply]
سلام محمدرضا. وقتت بخیر. 

فکر می‌کنم سؤال شما با مفهوم چیزی که من گفتم، چندان همخوانی نداره. ایده Zettelkasten به این معناست که کلاً از فولدر استفاده نکنیم و تگ هم تا حد امکان استفاده نشه. عملاً یک پوشه مسطح داشته باشیم که فایل‌ها به هم لینک داشته باشند. 

حالا با این لینک به مرور زمان یک گراف ساخته بشه و بتونیم دانش رو به این شکل ذخیره کنیم. 

برای هر شکلی از دانش هم قابل استفاده است؛ مثال بخوام بزنم من یک فایل به نام متمم دارم که همه نوت‌های متمم‌ام رو به اون لینک می‌دم. نوت‌های درس تصمیم‌گیری رو علاوه بر اون به یک فایل دیگه هم ارجاع میدم. حالا اگر خواستم نوت‌های تصمیم گیریم تو متمم رو بخونم نه از فایل استفاده می‌کنم و نه از پوشه. 

به یکی از فایل‌های تصمیم‌گیری می‌رم و با لینک فایل مدنظرم رو پیدا می‌کنم. در واقع ایده اصلی اینه که لینک از فولدر و تگ بسیار بسیار بهتر است. 

من تو سیستم خودم، خودم رو مجبور کردم که به هیچ وجه از دو سطح فولدر بیشتر استفاده نکنم ولی همون هم واقعاً لازم نیست. میشه همش رو بصورت مسطح نوشت. 

در واقع جذابیت ایده از اینجاست که از فایل‌ها همه داخل یک فولدر هستند و در عین این که به نظر بی‌نظم هستند یک نظم خوبی کم‌کم ازشون ظاهر میشه! در ضمن نوت‌ها رو هم میشه به هم ربط داد و به این یادگیری کریستالی بهتر انجام میشه. و چه چیزی بهتر از این. 

امیدوارم تونسته باشم جواب‌تون رو بدم.
